aries-test-harness/features/steps/0160-connection.py

```diff
@@ -372,7 +372,7 @@ def step_impl(context, sender, receiver):
     sender_connection_id = context.connection_id_dict[sender][receiver]
 
     data = {"comment": "acknowledgement from " + sender}
-    sleep(5)
+    sleep(1)
```

Comment:
Is it possible to use the polling loop in these case as we mentioned yesterday -- `wait_until(timeout, polling_interval)`, so in this case `wait_until(20,0.5)`.